Announce Virtual Comp, Mentoring, and Rules (#164)
srobo/team-emails: SR2024/2023-11-08-virtual-competition-and-mentoring.md

---
to: Student Robotics 2024
subject: SR2024 Virtual Competition
---

Hello! 

We hope your team found Kickstart useful and are excited for the year ahead - we certainly are, and we have a couple things to announce that will happen along the way:

# Virtual Competition

We're happy to announce that in addition to our physical competition, we'll be holding a **virtual competition**. We'll run our virtual competition on our [robot simulator](https://studentrobotics.org/docs/simulator/).

> The virtual competition is an opportunity for your teams to test their coding skills before the physical competition takes place.

Reviewer: https://github.com/srobo/team-emails/pull/164/commits/cfed3c7ad7ef5f0879dc4865a71fb0b85d6d2605, perhaps?

---
to: Student Robotics 2024

> subject: SR2024 Virtual Competition, Mentoring, and rules changes

Reviewer: ```suggestion
subject: SR2024 Virtual Competition, Mentoring, and Rules changes
```

---

Hello! 

> We hope your team found Kickstart useful and are excited for the year ahead - we certainly are, and we have a couple things to announce that will happen along the way:

Reviewer: ```suggestion
We hope your team found Kickstart useful and are excited for the year ahead - we certainly are, and we have a few things to announce that will happen along the way:
```

# Virtual Competition

We're happy to announce that in addition to our physical competition, we'll be holding a **virtual competition**. We'll run our virtual competition on our [robot simulator](https://studentrobotics.org/docs/simulator/).

The virtual competition is an opportunity for your teams to test their coding skills before the physical competition takes place.

The matches will be broadcast live on **Saturday 9th March 2024 at 1pm GMT**  via a [livestream](https://www.youtube.com/watch?v=hlfaQIfLaRg) on our YouTube channel.

> The code submission deadline will be **2 days before**, that's Thursday 7th March 2024 at 6pm GMT. Your team can submit their robot code to [the code submitter](https://studentrobotics.org/code-submitter/). The log in details have the username of your team's three letter acronym (in upper case): <TLA mailmerges in here>, and the password is the same as the password your team used to enter Discord.

Reviewer: ```suggestion
The code submission deadline will be **two days before**, that's Thursday 7th March 2024 at 6pm GMT. Your team can submit their robot code to [the code submitter](https://studentrobotics.org/code-submitter/). The log in details have the username of your team's three letter acronym (in upper case): <TLA mailmerges in here>, and the password is the same as the password your team used to enter Discord.
```

> During the competition, your robot will compete in (at least) 4 virtual matches for real league points, these points will then transfer across to the physical competition!

Reviewer 2: ```suggestion
During the competition, your robot will compete in a number of virtual matches for real league points, these points will then transfer across to the physical competition!
```

We'll keep the [virtual competition event page](https://studentrobotics.org/events/sr2024/virtual-competition) up-to-date with more information about the event.

We have made an amendment to our rules to add further details.

# Mentoring

This year, we'd like to offer mentoring support through the year. Mentoring support from one of our volunteers can be invaluable in terms of helping understand the kit, rules and competition; in encouraging and pushing your teams; and as enthusiastic role models.

Please fill out the form below to register an interest in mentoring. While we can't guarantee regular support, we'll do our best to match you with a volunteer as soon as we're able.

https://forms.gle/V831FNHaMLC4h4pVA

We would also like one of your students to post a short report in your team's Discord channel after they meet so we are better able to offer help. If you/your teams are not yet in [Discord](https://studentrobotics.org/docs/tutorials/discord.html) and you need help, please let us know.

# Rules amendments

> We've also made some minor amendments to the rules which are worth noting:

Reviewer: ```suggestion
We've also made some minor amendments to the [rules](https://srobo.org/rules/) which are worth noting:
```
